playhouse/pool.py

```diff
@@ -142,6 +147,12 @@ def _connect(self, *args, **kwargs):
     def _is_stale(self, timestamp):
         return (time.time() - timestamp) > self.stale_timeout
 
+    def _is_ok(self, conn):
+        # Return True if it's safe to return this connection to the pool.
+        # Retrun False otherwise.
+        # Implementing classes are encouraged to fix the connection (if possible).
+        return True
```

Comment:
The API should probably not be extended in this way. Let's change the name of "_is_closed" to "_is_usable" and keep things to a single method.